### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

It reported: https://github.com/jsoref/covidshield-server/commit/ae257405066988de942a692454958951f8f9d0d9#comments

And it validated that the changes in this PR made it happy: https://github.com/jsoref/covidshield-server/commit/58040ab9c35151c12181ca7b7512ba55bb838671

Note: this PR does not include the action, if you're interested, that can be offered separately.

### CODE REVIEW

File: pkg/telemetry/telemetry.go [hunk 1]
```diff
@@ -25,11 +25,11 @@ const PROMETHEUS = "prometheus"
 
 var log = logger.New("telemetry")
 
-type Cleanuper interface {
+type Cleaner interface {
```
Comment (burke) [inline pkg/telemetry/telemetry.go:28]: This one is (unfortunate but) idiomatic for the language.

Comment (jsoref) [reply pkg/telemetry/telemetry.go:28]: Sigh. I can drop it. Right now the commits are all signed (done via GitHub's UI). I'll probably be lazy and just do a rebase that'll strip that.

Fwiw, it doesn't appear particularly common (although admittedly it is part of golang/mock -- but I think that's a local, I wonder if I can convince them to change...).
https://www.google.com/search?sxsrf=ALeKk00FaKGX8zQO_JtUxhtNPh06JaNKOg:1592843353002&q=Cleanuper+golang+-php&nirf=Cleanup+golang+-php&sa=X&ved=2ahUKEwjd8_Px65XqAhWMSTABHQkiA7UQ8BYoAXoECA0QKA&biw=1280&bih=610&dpr=2.5

**Comment by jsoref** [comment]
If you'd like to prevent future errors from sneaking in, I'm happy to make a PR for the action. (One of the CovidShield repositories took it.)

File: pkg/telemetry/telemetry.go (hunk 1 above)
Comment (burke) [reply pkg/telemetry/telemetry.go]: Oh, it's not `Cleanuper` in specific, it's that single-method interfaces are suggested to be `<method-name>er`.

**Comment by burke** [comment]
Yeah I don't see any harm in adding an action for it! Thanks for the contribution

**Pull request merged**